arcade/application.py

```diff
@@ -30,6 +30,14 @@
 
 _window: 'Window'
 
+__all__ = [
+    "get_screens",
+    "NoOpenGLException",
+    "Window",
+    "open_window",
+    "View"
+]
```

Comment:
@einarf should this include the mouse button constants? I can see a user wanting to do the following:
```python
from arcade.application import *
```

Reply:
I've added this in https://github.com/gran4/arcade/pull/1 to get this PR moving again.